cryptostore/data/parquet.py

```diff
@@ -66,7 +66,7 @@ def write(self, exchange, data_type, pair, timestamp):
 
         if self._write:
             for func, bucket, prefix, kwargs in zip(self._write, self.bucket, self.prefix, self.kwargs):
-                path = f'{exchange}/{data_type}/{pair}/{int(timestamp)}.parquet'
+                path = f'{exchange}/{data_type}/{pair}/{exchange}-{data_type}-{pair}-{int(timestamp)}.parquet'
```

Comment:
I'll take this change, but you need to remove the changes unrelated to this issue from the PR.